docs/design/server_side_charts.md

```diff
@@ -0,0 +1,44 @@
+# Supporting charts on the server side
+As discussed in #367, the server side components manager and resourcifier have not yet been refactored, as of this writing, to work with [the new chart format](docs/design/chart_format.md). Presumably, expandybird should not be affected.
+
+## Context
+In discussions about the refactoring on the Slack channel, a number of potential requirements were discussed that could complicate matters, such as:
+* supporting multiple templating technologies within a single chart.
+* supporting multiple templating technologies within a single dependency graph.
```

Comment:
Well, currently, expandybird does take a set of files as input, but there's one file that serves as the top level template. All of the other files are imports.

Without a single top level file, I'm not sure how it could do its work. Currently, it's quite straight forward. Process the top level file as a list of resources, each of which is either a primitive type or a template reference. If it's a template reference, expand it.